.github/workflows/phpstan.yml

```diff
@@ -3,10 +3,10 @@ on: [push, pull_request]
 jobs:
   phpstan:
     name: phpstan
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
We don't need to do this, this will be done automatically in a few weeks (see https://github.com/actions/virtual-environments/issues/1816) unless we want to freeze the environment version

Reply:
The problem is that all Github Actions for PHPStan are locked. (some with multiples restarts).

Reply:
Ah, you believe there is an issue with `ubuntu-latest` and you think using `20.04` can solve it?

Reply:
It's the case. But we can wait a few weeks, if you think it's better.

Reply:
We can try :) and anyways it's better to use fixed OS versions as it improves reproductibility